project-base/templates/Front/Inline/Cart/cartBox.html.twig

```diff
@@ -1,6 +1,15 @@
-<div id="js-cart-box" data-reload-url="{{ url('front_cart_box') }}">
+{% import 'Front/Inline/Cart/cartBoxItemMacro.html.twig' as cartBoxItem %}
+
+<div id="js-cart-box"
+    {% if cart is not null and getRoute() not in pagesWithDisabledCartHover %}
+        class="js-hover-intent{% if isIntentActive is same as('true') %} active{% endif %}"
+        data-hover-intent-class-for-open="active"
+    {% endif %}
+    data-reload-url="{{ url('front_cart_box') }}"
+>
     <div class="cart">
-        <a class="cart__block" href="{{ url('front_cart') }}">
+        <a {% if cart is not null %}href="{{ url('front_cart') }}"{% endif %}
```

Comment:
In HTML5 anchor without href is valid and it's considered placeholder link. This is new information to me :) nice 👍